tenzoMath/fanucModel.cpp

```diff
@@ -1,80 +1,41 @@
-#include "fanucModel.h"
-#include <iostream>
+#include "FanucModel.h"
+
+#include "poly34.h"
 #include <opencv2/core.hpp>
+
+#include <iostream>
 #include <vector>
-#include "poly34.h"
 
-FanucModel::FanucModel()
-    : RoboModel(std::vector<std::array<double, 4>>{
-          {0, 0, 150, PI / 2},
-          {0, 0, 790, 0},
-          {0, 0, 250, PI / 2},
-          {835, 0, 0, -PI / 2},
-          {0, 0, 0, PI / 2},
-          {100, 0, 0, 0},
-          {130, PI / 2, -90, 0},
-          {-190, 0, 0, 0}
-      }),
-      _toCamera((cv::Mat_<double>(4, 4) << 0, -1, 0, -43, 1, 0, 0, -90, 0, 0, 1, 130, 0, 0, 0, 1)),
-      _toSixth((cv::Mat_<double>(4, 4) << 1, 0, 0, 0, 0, 1, 0, 0, 0, 0, 1, -190, 0, 0, 0, 1)),
-      _forMovingToCamera((cv::Mat_<double>(4, 4) << 0, -1, 0, 43, 1, 0, 0, 90, 0, 0, 1, 130, 0, 0, 0, 1))
+namespace nikita
 {
-}
-
-std::vector<RoboModel::DhParameters> FanucModel::getDhParameters() const
+FanucModel::FanucModel()
+     : RoboModel(std::vector<std::array<double, 4>>{
+         {0, 0, 150, PI / 2},
+         { 0, 0, 790, 0 },
+         { 0, 0, 250, PI / 2 },
+         { 835, 0, 0, -PI / 2 },
+         { 0, 0, 0, PI / 2 },
+         { 100, 0, 0, 0 },
+         { 130, PI / 2, -90, 0 },
+         { -190, 0, 0, 0 }
+})
+{}
+
+std::vector<double> FanucModel::jointsToQ(const std::array<double, 6>& j)
 {
-    return _kinematicChain;
+    return { {
```

Comment:
**(note, repeatedly)**: no obligation to write an additional pair of braces inside now.